.coderabbitai.yaml

```
@@ -0,0 +1,20 @@
+    # Set to true if you want it to review Draft PRs
+    draft_prs: false 
+    # Set to true if you want it to review PRs from forks
+    base_branches:
+      - main
+      - master
+
+chat:
+  auto_reply: true
+
+# Optional: Ignore specific files
+reviews:
```

YAMLlint (1.37.1)

[error] duplication of key "reviews" in mapping

(key-duplicates)